react/features/unsupported-browser/components/NoMobileApp.js

```diff
@@ -0,0 +1,34 @@
+/* @flow */
+
+import React, { Component } from 'react';
+
+declare var interfaceConfig: Object;
+
+/**
+ * React component representing no mobile page.
+ *
+ * @class NoMobileApp
+ */
+export default class NoMobileApp extends Component {
+
+    /**
+     * Renders the component.
+     *
+     * @returns {ReactElement}
+     */
+    render() {
+        const ns = 'no-mobile-app';
+
+        return (
+            <div className = { ns }>
+                <h2 className = { `${ns}__title` }>
+                    Video chat isn't available in the mobile apps
+                </h2>
+                <p className = { `${ns}__description` }>
+                    Please use { interfaceConfig.APP_NAME } on <br />
+                    Desktop top join calls.
+                </p>
```

Comment:
Let's change this message to: "Video chat isn't available on mobile" and then fix the type in the second line "Desktop to join calls".

Also it doesn't seem like a good idea to add a br tag in the second paragraph, instead we should fix that with applying the appropriate css that aligns the text correctly and leaves the spacing we want.